src/plugins/Maps/MapsPlugin.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import Maps from "./Maps";
+
+class Map extends Component{
```

Comment:
why is this class called `Map` and the file `MapsPlugin`? Please consider a consistent naming, e.g. `MapPlugin` for both or similar